tusclient/uploader.py

```diff
@@ -71,24 +84,34 @@ class Uploader(object):
         - metadata (Optional[dict])
         - retries (Optional[int])
         - retry_delay (Optional[int])
+        - resumable (Optional[bool])
+        - storage (Optinal [<tusclient.storage.interface.Storage>])
+        - fingerprinter (Optional [<tusclient.fingerprint.interface.Fingerprint>])
     """
     DEFAULT_HEADERS = {"Tus-Resumable": "1.0.0"}
     DEFAULT_CHUNK_SIZE = 2 * 1024 * 1024  # 2MB
 
     def __init__(self, file_path=None, file_stream=None, url=None, client=None,
-                 chunk_size=None, metadata=None, retries=0, retry_delay=30):
+                 chunk_size=None, metadata=None, retries=0, retry_delay=30,
+                 resumable=False, storage=None, fingerprinter=None):
         if file_path is None and file_stream is None:
             raise ValueError("Either 'file_path' or 'file_stream' cannot be None.")
 
         if url is None and client is None:
             raise ValueError("Either 'url' or 'client' cannot be None.")
 
+        if resumable and storage is None:
+            raise ValueError("Please specify a storage instance to enable resumablility.")
```

Comment:
The documentation above said that a default storage would be used here:
> if resumable is set to true, the default storage module (<tusclient.storage.filestorage.FileStorage>) would be used.

Reply:
yes, that's wrong. I should remove it from the documentation.